multisite-directory.php

```diff
@@ -49,7 +50,9 @@ public static function register () {
         add_action('delete_blog', array(__CLASS__, 'delete_blog'), 10, 2);
         add_action('network_admin_menu', array('WP_Multisite_Directory_Admin', 'network_admin_menu'));
         add_action('signup_blogform', array(__CLASS__, 'signup_blogform'));
-        add_action('add_signup_meta', array(__CLASS__, 'add_signup_meta'));
+        add_action('network_site_new_form', array(__CLASS__, 'signup_blogform'));
```

Comment:
I added a check using the `$pagenow` global to detect whether the user is on the network admin screen and output the additional HTML. Not sure if you're gonna be happy with that approach though, as your suggestion would surely be more elegant. I also added a docblock `@uses` to document the usage of the global variable - please feel free to adjust the language if it doesn't feel right, I'm not actually fluent in English.

I think we could get rid of the list bullets (they were already visible on the front-end sign-up form). Do you think injecting an inline style in the checklist `$html` would be too smelly? :)